Doc/library/ssl.rst

```diff
@@ -1878,7 +1878,7 @@ to speed up repeated connections from the same clients.
 
 .. attribute:: SSLContext.check_hostname
 
-   Whether to match the peer cert's hostname with :func:`match_hostname` in
+   Whether to match the peer cert's hostname in
```

Comment:
It might be better to re-flow this text to avoid this awkward hanging text.

Reply:
Sure, I didn't do it to make the review easier. Will do now.